### PR TITLE
Added EAK's Cookie Bear program (EAK;TS BEAR).

### PR DESCRIPTION
**Pull request by eswenson1**
This version actually works.

### CODE REVIEW

File: build/misc.tcl [hunk 1]
```diff
@@ -695,6 +695,10 @@ respond "SUBJECT:" "COOKIE\r"
 respond "NAME:" "BEAR\r"
 expect ":KILL"
 
+# Cookie Bear (this one actually works)
+respond "*" "midas eak;ts bear_eak;bear\r"
```
Comment (larsbrinkhoff) [inline build/misc.tcl:699]: This is missing a colon.

**Review by larsbrinkhoff — CHANGES_REQUESTED** [review]
Please add file timestamps.

**Comment by larsbrinkhoff** [comment]
Suggest adding to the commit comment some description of how to interact with the program.

**Comment by eswenson1** [comment]
I've added timestamps and updated the commit log with information on how to invoke BEAR. I also fixed the missing ":" in the TCL script.  @larsbrinkhoff Please let me know when you approve and I can merge.

**Comment by eswenson1** [comment]
@larsbrinkhoff This PR still says that there are requested changes that aren't yet resolved.  How do I indicate that I resolved all the issues you brought up?  Or is this something you have to do?

**Review by larsbrinkhoff — APPROVED** [review]
Looks nice now!

**Comment by larsbrinkhoff** [comment]
I think they are resolved by someone clicking approve.

**Comment by larsbrinkhoff** [comment]
All good here.

**Pull request merged**